extensions/Worker.Extensions.Storage.Blobs/src/BlobStorageConverter.cs

```diff
@@ -14,12 +14,24 @@
 using Microsoft.Azure.Functions.Worker.Core;
 using Microsoft.Extensions.Options;
 using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
 namespace Microsoft.Azure.Functions.Worker
 {
     /// <summary>
     /// Converter to bind Blob Storage type parameters.
     /// </summary>
+    [SupportsDeferredBinding]
+    [SupportsJsonDeserialization]
+    [SupportedConverterTypes(typeof(string), supportsCollection: true)]
+    [SupportedConverterTypes(typeof(Stream), supportsCollection: true)]
+    [SupportedConverterTypes(typeof(byte[]), supportsCollection: true)]
+    [SupportedConverterTypes(typeof(BlobBaseClient), supportsCollection: true)]
+    [SupportedConverterTypes(typeof(BlobClient), supportsCollection: true)]
+    [SupportedConverterTypes(typeof(BlockBlobClient), supportsCollection: true)]
+    [SupportedConverterTypes(typeof(PageBlobClient), supportsCollection: true)]
+    [SupportedConverterTypes(typeof(AppendBlobClient), supportsCollection: true)]
+    [SupportedConverterTypes(typeof(BlobContainerClient), supportsCollection: true)]
```

Comment:
I know we technically support this, but its not a valid scenario

Reply:
I did not follow. Which scenario are we talking about?

Reply:
ohh got it, you mean `BlobContainerClient`. I will remove it.

Reply:
For trigger we will need Container client - https://github.com/Azure/azure-functions-dotnet-worker/blob/cb93c716190b75dea3d85edb4344b06a156ee267/test/E2ETests/E2EApps/E2EApp/Blob/BlobTriggerBindingFunctions.cs#L75

Reply:
Resolving it as design is updated here and we won't advertise specific types for this converter now.